src/App/Pages/Settings/SettingsPage/SettingsPageViewModel.cs

```diff
@@ -286,6 +288,27 @@ public async Task VaultTimeoutAsync(bool promptOptions = true, int? newTimeout =
             }
         }
 
+        public async Task LoggerReportingAsync()
+        {
+            _reportLoggingEnabled = await _loggerService.IsEnabled();
```

Comment:
Here you don't need to have this line `_reportLoggingEnabled = await _loggerService.IsEnabled();` you're already updating `_reportLoggingEnabled` in line 308 so you don't need to do this again here unless I'm mistaken and there is some flow in which it's outdated. If so, let me know because I can't think of any in which line 293 is necessary

Reply:
I let that call stay just in case. But i think we can remove it.